bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -62,6 +74,34 @@ TEST_F(InstanceAdminIntegrationTest, ListInstancesTest) {
   }
 }
 
+/// @test Verify that InstanceAdmin::DeleteInstances works as expected.
+TEST_F(InstanceAdminIntegrationTest, DeleteInstancesTest) {
+  // The emulator does not support instance operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+  std::string instance_id = "DeleteInstancesTest_instance";
+  bigtable::InstanceId instance(instance_id);
+  bigtable::DisplayName diplay_name(instance_id);
+  EXPECT_FALSE(
+      IsInstancePresent(instance_admin_->ListInstances(), instance_id));
+
+  std::vector<std::pair<std::string, bigtable::ClusterConfig>> clusters;
+  clusters.push_back(std::make_pair(
+      "sample-cluster",
+      bigtable::ClusterConfig(
+          "sample-cluster", 1234,
```

Comment:
What is `1234` here?

Reply:
Ah, the number of nodes I think.  Please change this to create a development instance, production instances require at least 3 nodes and they are expensive (and slower) to create.

Reply:
changed it to 1.